packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -532,8 +532,31 @@ class _MonthPickerState extends State<MonthPicker> {
     });
   }
 
+  Icon _getPreviousMonthIcon(TextDirection textDirection) {
+    assert(textDirection != null);
+    switch (textDirection) {
+      case TextDirection.rtl:
+        return const Icon(Icons.chevron_right);
+      case TextDirection.ltr:
+        return const Icon(Icons.chevron_left);
```

Comment:
so these don't mirror?

Reply:
I didn't see a `chevron_forward` or anything that would mirror.